src/Components/TableHeader/TableHeader.js

```diff
@@ -1,29 +1,42 @@
 import sortArrow from '../../Assets/Icons/sort-24px.svg'
-
 import './TableHeader.scss'
+import React,{useEffect, useRef} from 'react'
+
+const TableHeader = (props) =>{  
+
+    const fifthHeader = useRef([])
```

Comment:
No change requested as it works - I do see one way Niko's suggestion could be implemented, where it would be passing in an object where the key is the column name, and the value is the width percentage, then in the table header, we would generate the columns dynamically (similar to how you did rows).

Reply:
Sounds like a good idea! Perhaps we can do that when the site is finished?